wazo_webhookd/services/mobile/plugin.py

```diff
@@ -98,8 +102,33 @@ def on_external_auth_deleted(self, body, event):
                 self.subscription_service.delete(subscription.uuid)
             logger.info('User unregistered: %s/%s', tenant_uuid, user_uuid)
 
+    @classmethod
+    def get_subscription_token(cls, config):
+        if config["mobile_use_subscription_token"] and (
+            not cls._subscription_token or not cls._is_valid(cls._subscription_token)
```

Comment:
This validation will occur each time we send a push notification, even if we send it through FCM. APNS is only needed for `incomingCall` for ios devices; all other push notifications go through FCM. Also, if APNS is not even configured on the platform, webhookd will still try to get the subscription token, even though it will never need it. So I think we should delay the subscription validation until we're sure we actually need it.